File: app/src/main/java/com/softhink/single/models/AppInfo.java
package com.softhink.single.models;

import android.os.Build;

import com.google.gson.annotations.SerializedName;
import com.softhink.single.BuildConfig;

public class AppInfo {
    @SerializedName("versionOs")
    private String versionOs;
    @SerializedName("versionApp")
    private String versionApp;
    @SerializedName("os")
    private Integer os;

    public AppInfo() {
        this.versionOs = String.valueOf(Build.VERSION.SDK_INT);
        this.versionApp = BuildConfig.VERSION_NAME;
        this.os = 1;
    }
}

File: app/src/main/java/com/softhink/single/data/model/AppInfo.kt
package com.softhink.single.data.model

import android.os.Build

import com.google.gson.annotations.SerializedName
import com.softhink.single.BuildConfig

class AppInfo {
    @SerializedName("versionOs")
    private val versionOs: String
    @SerializedName("versionApp")
    private val versionApp: String
    @SerializedName("os")
    private val os: Int?

    init {
        this.versionOs = Build.VERSION.SDK_INT.toString()
        this.versionApp = BuildConfig.VERSION_NAME
        this.os = 1
    }
}
